google/cloud/bigquery/client.py

```diff
@@ -1976,15 +1976,8 @@ def create_job(
             )
             destination = _get_sub_prop(job_config, ["copy", "destinationTable"])
             destination = TableReference.from_api_repr(destination)
-            sources = []
```

Comment:
There was a unit test failing because we were actually transforming `"sourceTable"` to `"sourceTables"` without this change.